nifty-examples/src/test/java/com/facebook/nifty/server/DeafSelectorProvider.java

```diff
@@ -68,6 +68,7 @@ public AbstractSelector openSelector() throws IOException
     {
         return new AbstractSelector(this) {
 
+            private final Object lock = new Object();
```

Comment:
I understand the old implementation was dummy, but is the new impl required for the version bump, or it's not related?